pallets/statistics/src/benchmarking.rs

```diff
@@ -34,13 +34,23 @@ fn init_ticker<T: Trait>() -> (User<T>, Ticker) {
 
 fn init_ctm<T: Trait>() -> (User<T>, Ticker, Vec<TransferManager>) {
     let (owner, ticker) = init_ticker::<T>();
-    let tms = (0..T::MaxTransferManagersPerAsset::get())
+    let tms = (1..T::MaxTransferManagersPerAsset::get())
```

Comment:
Does it matter ?

Reply:
Yeah, I am confused about this as well. This shouldn't matter.

Reply:
From [this line](https://github.com/PolymathNetwork/Polymesh/blob/f58ec62d9c4a843099fb67bec4ef1a6238d4b58f/pallets/statistics/src/lib.rs#L106), we need strict less value, so this change generates (`MaxTransferManagerPerAsset -1`)

at. @satyamakgec  @maxsam4

Reply:
@fmiguelgarcia but that line checks the length before pushing the last tm. i.e. If you are pushing the `MaxTransferManagerPerAsset` module, the length in that function should be `MaxTransferManagerPerAsset - 1`

Reply:
New implementation which fixes the specific `add_transfer_manager`. 
It is cleaner than previous one because now the change is inside `add_transfer_manager`, nor  in `init_ctm` and it does not affect to `remove_transfer_manager`.

Pls, take a look: @maxsam4 @satyamakgec